README.md

```diff
@@ -427,8 +396,7 @@ end
 ```
 
 We have defined a constant `Outcome` in the class of type `Struct`. All outcome
-values are `Struct`s so they are uniformly structured. A `Struct` is essentially
-a list of named values.
+values are `Struct`s. A `Struct` is essentially a list of named values.
```

Comment:
Order does matter with structs, though:

```ruby
Foo = Struct(:a, :b, :c)

val = Foo.new('two, 'three', 'one')
puts val.a # => 'two'
```

It's essential with SJ to realize that the ordering is important, because we deconstruct them using splatting, which requires order. That's why I framed it this way.